### PR TITLE
Add a maker for creating files with code for Code objects

### PR DESCRIPTION
**Pull request by nbirillo**

### CODE REVIEW

File: src/test/solution_space/solution_graph/code_to_file_tests.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+
+    # Create three graphs and check all folders names which was created for each graph
+    def test_folders_names(self):
+        delete_folder()
+        init_default_ids()
+        sg_0, sg_1, sg_2 = create_three_graphs()
+        expected_folders_names = get_full_paths([GRAPH_FOLDER_PREFIX + '0',
+                                                 GRAPH_FOLDER_PREFIX + '1',
+                                                 NOT_DEFAULT_GRAPH_PREFIX + '2'])
+        self.assertTrue(compare_two_lists_of_names(expected_folders_names, get_actual_folders_names()))
```
Comment (elena-lyulina) [inline src/test/solution_space/solution_graph/code_to_file_tests.py]: For comparing two lists of strings you can use self.assertCountEqual with two lists as arguments (therefore you can remove compare_two_lists_of_names method)

File: src/main/solution_space/consts.py [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
 
 class VERTEX_TYPE(Enum):
     START = 'start'
     END = 'end'
     INTERMEDIATE = 'intermediate'
+
+
+FOLDER_WITH_CODE_FILES = ROOT_DIR + '/../../resources/solution_space'
+FOLDER_WITH_CODE_FILES_FOR_TESTS = ROOT_DIR + '/../../resources/solution_space_tests'
```
[the remark this answers is not in the document]
Comment (nbirillo) [reply src/main/solution_space/consts.py]: A good idea!

File: src/main/solution_space/solution_graph.py [hunk 3]
```diff
@@ -2,27 +2,38 @@
...
+                 vertex_type: solution_space_consts.VERTEX_TYPE = solution_space_consts.VERTEX_TYPE.INTERMEDIATE):
         self._parents = []
         self._children = []
         self._code_info_ist = []
         self._code = code
         self._vertex_type = vertex_type
 
+        if code:
+            if not is_exist(graph.graph_directory):
+                log.error(f'The graph with id {graph.id} does not have directory for code for vertices. Expected graph '
```
[the remark this answers is not in the document]
Comment (nbirillo) [reply src/main/solution_space/solution_graph.py]: Thank you for the correction of my English!

File: src/main/util/language_util.py [hunk 4]
```diff
@@ -1,14 +1,15 @@
 # Copyright (c) 2020 Anastasiia Birillo, Elena Lyulina
 
+from typing import Optional
 from src.main.util import consts
 
 
-def get_language_by_extension(extension: str):
+def get_language_by_extension(extension: str) -> consts.LANGUAGE:
     return consts.EXTENSION_TO_LANGUAGE_DICT.get(extension, consts.LANGUAGE.NOT_DEFINED.value)
```
[the remark this answers is not in the document]
Comment (nbirillo) [reply src/main/util/language_util.py]: I did it in another pr :)

**Pull request merged**